dev/reference/tools/tidb-lightning/config.md

```diff
@@ -0,0 +1,324 @@
+---
+title: TiDB Lightning Configuration
+summary: Learn about the CLI usage and sample configuration in TiDB Lightning.
+category: reference
+---
+
+# TiDB Lightning Configuration
+
+This document provides samples for global configuration, task configuration, and TiKV Importer configuration in TiDB Lightning, and describes the usage of command-line parameters.
+
+## Configuration files
+
+TiDB Lightning's global and task configurations have compatible structures. Unless [Server mode](/dev/reference/tools/tidb-lightning/web.md) is enabled, the same file is used for both parts of configuration.
```

Comment:
Hi @lilin90, I have a question to ask you, is this description clear to you?
```
Unless [Server mode](/dev/reference/tools/tidb-lightning/web.md) is enabled, the same file is used for both parts of configuration.
```

Reply:
@kennytm 
- Do you mean "the same `toml` configuration file" by "the same file"? If yes, what's this file like?
- How about `The same configuration file is used for both global and task configurations unless the [server mode](/dev/reference/tools/tidb-lightning/web.md) is enabled.`
- I think it would be more friendly if you can add some description about when the user should enable this server mode.